docs/zh_CN/API.md

```diff
@@ -0,0 +1,1005 @@
+    console.log('End. Total size = ' + size)
+  })
+  dataStream.on('error', function(err) {
+    console.log(err)
+  })
+})
+```
+
+<a name="fGetObject"></a>
+### fGetObject(bucketName, objectName, filePath[, callback])
+
+下载并将对象保存成本地文件。
+
+__参数__
+
+| 参数  |  类型 | 描述  |
+|---|---|---|
+| `bucketName`  | _string_   | 存储桶名称。  |
+| `objectName`  |_string_   | 对象名称。  |
+| `filePath`  |  _string_ | 要写入的本地文件路径。  |
+| `callback(err)`  | _function_  | 如果报错的话，则会调用回调函数，传入`err`参数。 如果没有传callback的话，则返回一个`Promise`对象。 |
+
+
+__示例__
+
+
+```js
+var size = 0
+minioClient.fGetObject('mybucket', 'photo.jpg', '/tmp/photo.jpg', function(err) {
+  if (err) {
+    return console.log(err)
+  }
+  console.log('success')
+})
+```
+<a name="putObject"></a>
+### putObject(bucketName, objectName, stream, size, contentType[, callback])
+
+从一个stream/Buffer中上传一个对象。
+
+##### 从stream中上传
+
+__参数__
+
+
+| 参数  |  类型 | 描述  |
+|---|---|---|
+| `bucketName`  |_string_   | 存储桶名称。  |
+| `objectName`  |_string_   | 对象名称。  |
+| `stream`  | _Stream_  |可以读的流。   |
+|`size`   | _number_  | 对象的大小（可选）。  |
+|`contentType`   | _string_  | 对象的Content-Type（可选，默认是`application/octet-stream`）。|
+| `callback(err, etag)` | _function_ | 如果`err`不是null则代表有错误，`etag` _string_是上传的对象的etag值。如果没有传callback的话，则返回一个`Promise`对象。 |
+
+
+__示例__
+
+单个对象的最大大小限制在5TB。putObject在对象大于5MiB时，自动使用multiple parts方式上传。这就当上传失败时，客户端只需要上传未成功的部分即可（类似断点上传）。上传的对象使用MD5SUM签名进行完整性验证。
+
+```js
+var Fs = require('fs')
+var file = '/tmp/40mbfile'
+var fileStream = Fs.createReadStream(file)
+var fileStat = Fs.stat(file, function(err, stats) {
+  if (err) {
+    return console.log(err)
+  }
+  minioClient.putObject('mybucket', '40mbfile', fileStream, stats.size, function(err, etag) {
+    return console.log(err, etag) // err should be null
+  })
+})
+```
+
+##### 从"Buffer"或者"string"上传
+
+__参数__
+
+
+| 参数  |  类型 | 描述  |
+|---|---|---|
+| `bucketName`  |_string_   | 存储桶名称。  |
+| `objectName`  |_string_   | 对象名称。  |
+|`string or Buffer`   | _Stream_ or _Buffer_  |字符串可者缓冲区   |
+| `contentType`  | _string_   | 对象的Content-Type（可选，默认是`application/octet-stream`）。 |
+| `callback(err, etag)`  | _function_  |如果`err`不是null则代表有错误，`etag` _string_是上传的对象的etag值。 |
+
+
+__示例__
+
+
+```js
+var buffer = 'Hello World'
+minioClient.putObject('mybucket', 'hello-file', buffer, function(err, etag) {
+  return console.log(err, etag) // err should be null
+})
+```
+<a name="fPutObject"></a>
+### fPutObject(bucketName, objectName, filePath, contentType[, callback])
+
+上传文件。
+
+__参数__
+
+
+| 参数  |  类型 | 描述  |
+|---|---|---|
+| `bucketName`  | _string_  | 存储桶名称。  |
+|`objectName`   |_string_   | 对象名称。  |
+| `filePath`  | _string_  | 要上传的文件路径。  |
+| `contentType`  | _string_  | 对象的Content-Type。  |
+| `callback(err, etag)`  |  _function_ | 如果`err`不是null则代表有错误，`etag` _string_是上传的对象的etag值。如果没有传callback的话，则返回一个`Promise`对象。 |
+
+__示例__
+
+
+单个对象的最大大小限制在5TB。fPutObject在对象大于5MiB时，自动使用multiple parts方式上传。这就当上传失败时，客户端只需要上传未成功的部分即可（类似断点上传）。上传的对象使用MD5SUM签名进行完整性验证。
```

Comment:
ditto

Reply:
> ditto
@yihuaf  The original docs is ditto too,but I agree with you,it's a little redundant.I removed it in Chinese docs.